awx/main/tests/docs/test_swagger_generation.py

```diff
@@ -7,7 +7,7 @@
 from django.utils.functional import Promise
 from django.utils.encoding import force_str
 
-from openapi_codec.encode import generate_swagger_object
+from drf_yasg.codecs import OpenAPICodecJson
```

Comment:
@shanemcd @AlanCoding I got the remaining tests working on this PR but was hoping one of you could take a look at how I changed this test to make it work and give a sign off on it.